packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -29,23 +29,49 @@ const propTypes = {
    */
   title: PropTypes.string,
   /**
-   * Message of the notification-dialog.
+   * @private Message of the notification-dialog
```

Comment:
Can we be upfront about these deprecated props on our props table? I'd rather we highlight these as `deprecated and/or will remove on next mvb` instead of hiding them from the props table. Makes it a little easier for consumers to follow

Reply:
Marked them as `Deprecated` here https://github.com/cerner/terra-framework/pull/899/commits/71070f993bf783fb6729629dfd5d55cd5044cc83